### DISCUSSION TITLE
tts-az-1是503，tts-1是400

### DISCUSSION THREAD

**Discussion by daifai** [BurnHair/docs#138] [技术支持]
tts-az-1是503，tts-1是400

{
  "error": {
    "message": "当前分组 default 下对于模型  无可用渠道 (request id: x)",
    "type": "one_api_error"
  }
}

**Comment by BennyThink** [comment]
接口没问题，报错信息中模型名字都没有显示出来，所以是你请求的方式不正确。

参考文档进行使用
https://github.com/BurnHair/docs#13-tts
